### PR TITLE
MODULES-6635: add option to pass SSLCertificateFile and SSLCertificateKey to ssl.conf

### PR DESCRIPTION
**Pull request by sudodevnull**
This will allow for the option to use mod_ssl with cert file and key if/when vhosts are not defined. Referenced ticket : [MODULES-6635](https://tickets.puppetlabs.com/browse/MODULES-6635)

### CODE REVIEW

File: manifests/mod/ssl.pp [hunk 1]
```diff
@@ -3,6 +3,8 @@
   $ssl_cryptodevice                                         = 'builtin',
   $ssl_options                                              = [ 'StdEnvVars' ],
   $ssl_openssl_conf_cmd                                     = undef,
+  $ssl_cert                                                 = undef,
```
Comment (ekohl) [inline manifests/mod/ssl.pp]: Should this be ```Optional[String] $ssl_cert``` or even ```Optional[Stdlib::Absolutepath] $ssl_cert```? Same for ssl_key.

Comment (sudodevnull) [reply manifests/mod/ssl.pp]: updated.

**Comment by HelenCampbell** [comment]
Thanks @sudodevnull ! Merging.

**Comment by sudodevnull** [comment]
QQ, when will the version be updated so that I may use the new feature?

**Comment by HelenCampbell** [comment]
Probably some time within the next month, the modules due a conversion with PDK and it will be released straight after :)

**Comment by sudodevnull** [comment]
Any better ETA at this point?  I would prefer not to pull down the apache module locally and make modifications to include this feature.  
Management is pushing for an ETA for me to be able to apply our apache module to production and I need this feature in order to not break things.

**Comment by HelenCampbell** [comment]
Apologies! I'm literally in the middle of one at the moment, it should be out momentarily 👍 @sudodevnull

**Comment by sudodevnull** [comment]
Awesome thank you!

**Pull request merged**